phpunit.xml.dist

```diff
@@ -1,25 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         bootstrap="tests/bootstrap.php"
->
-    <testsuites>
-        <testsuite name="Pheanstalk Test Suite">
-            <directory>./tests</directory>
-        </testsuite>
-    </testsuites>
-
-    <filter>
-        <whitelist>
-            <directory>./src</directory>
-        </whitelist>
-    </filter>
-
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" backupGlobals="false" backupStaticAttributes="false" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false" bootstrap="tests/bootstrap.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage>
```

Comment:
This is the PHPUnit 9.3 way of Configuring coverage.

It will _only_ work with PHPUnit 9.3+ which will limit the PHP versions from which coverage can be obtained. The "old style" configuration will continue to work until PHPUnit 10 and PHPUnit offers a command line way to "on the fly" update from the "old style" configuration to "new style", so if it is important to be able to retrieve code coverage for older PHP versions, I'd recommend using the "old style" configuration for the time being and if needs be, to update it on the fly to the "new style".

Reply:
I reverted this change.